views/layout.hbs

```diff
@@ -10,7 +10,13 @@
   </head>
 
   <body>
-
+    <li id="to-right">
```

Comment:
You could use a bootstrap navbar to make it easier and train bootstrap a little. Overall, everything looks as it should!